helm/cluster/templates/bastion/_helpers_files.tpl

```diff
@@ -0,0 +1,14 @@
+{{- define "cluster.internal.bastion.kubeadm.files" }}
```

Comment:
Template that renders files spec in kubeadm for bastion machines. There is just SSH in there, as these machines do not become Kubernetes nodes.